### DISCUSSION TITLE
How can I get sellected Cell name onContextMenuClick

### DISCUSSION THREAD

**Discussion by kenyazilim** [vitmalina/w2ui#2360] [Q&A] [answered]
I'm using version 2 and I have

```
,onClick: function (event) {
            let seciliRow = event.detail.recid;
            let seciliCell = event.detail.column;
            console.log("onClick seciliCell : " + seciliCell);
}
```

in my code but

```
,onContextMenuClick: function (target, event) {
            seciliRow = event.detail.recid;
            seciliCell = event.detail.column;
            console.log("onClick onContextMenuClick : " + seciliCell);
 }
```

is not working.

How can I get selected cell onContextMenuClick

**Comment by vitmalina** [comment]
I fixed the event to include `column` when click event is generated, but for now, you can use this code
```js
let td = query(event.originalEvent.target).closest('td')
let column = parseInt(td.attr('col'))
```

**Reply by kenyazilim** [reply]
onContextMenuClick event has no originalEvent. 

```
for (const key in event) {
    console.log(`event.${key}: ${event[key]}`);
}
```

even this code dos not shows originalEvent as an object in event

**Reply by vitmalina** [reply, marked as answer]
My bad, it is actually in `event.detail.originalEvent` Here is an example with it https://w2ui.com/web/demos/#/grid/37

**Reply by vitmalina** [reply]
I also adde column parameter to the event, see https://github.com/vitmalina/w2ui/commit/7a05e755e97d5cebff363129590105a93ff137c7
